pkg/cinder/mount/mount.go

```diff
@@ -25,13 +25,13 @@ import (
 	"k8s.io/kubernetes/pkg/util/mount"
 	"k8s.io/kubernetes/pkg/volume/util"
 	utilexec "k8s.io/utils/exec"
-	
```

Comment:
we need to follow up with gofmt checker in CI

Reply:
+1 Yes. I am doing it manually now.